README.md

```diff
@@ -381,9 +381,7 @@ use-dart-handle: true
 
 ```yaml
 preamble: |
-  /// AUTO GENERATED FILE, DO NOT EDIT.
-  ///
-  /// Generated by `package:ffigen`.
```

Comment:
Probably also keep this. It's a good habit for people to do this.

Reply:
Actually, we are writing this for every generated file, and the user cannot remove this.
So using this example for the preamble was _kind of_ misleading.
https://github.com/dart-lang/ffigen/blob/98b2d4ff29a8f054dce52a30b909f423d4cbd567/lib/src/code_generator/writer.dart#L159-L161

Reply:
Awesome!